core/services/postgres/connection.go

```diff
@@ -0,0 +1,72 @@
+package postgres
+
+import (
+	"time"
+
+	"github.com/pkg/errors"
+	uuid "github.com/satori/go.uuid"
+	"github.com/scylladb/go-reflectx"
+
+	"github.com/smartcontractkit/chainlink/core/gracefulpanic"
+	"github.com/smartcontractkit/chainlink/core/logger"
+	"github.com/smartcontractkit/chainlink/core/store/config"
+
+	"github.com/smartcontractkit/sqlx"
+	gormpostgres "gorm.io/driver/postgres"
+	"gorm.io/gorm"
+	"gorm.io/gorm/clause"
+
+	// We've specified a later version in go.mod than is currently used by gorm
+	// to get this fix in https://github.com/jackc/pgx/pull/975.
+	// As soon as pgx releases a 4.12 and gorm [https://github.com/go-gorm/postgres/blob/master/go.mod#L6]
+	// bumps their version to 4.12, we can remove this.
```

Comment:
is there a story to track?

Reply:
I think this is obsolete, I copied it verbatim from https://github.com/smartcontractkit/chainlink/blob/3fc7397b49d4b5b9a03c3fdf46e3bc256064e508/core/store/orm/orm.go#L24-L28

Reply:
pgx updated to 4.13 in #5061

Reply:
And go-gorm/postgres updated to 4.13 @ v1.1.1 ten days ago: https://github.com/go-gorm/postgres/blob/v1.1.1/go.mod